tokio/src/runtime/builder.rs

```diff
@@ -728,6 +728,7 @@ impl Builder {
     /// # }
     /// ```
     #[cfg(all(not(loom), tokio_unstable))]
+    #[cfg_attr(docsrs, doc(cfg(all(not(loom), tokio_unstable))))]
```

Comment:
Loom should not be mentioned in docs.
```suggestion
    #[cfg_attr(docsrs, doc(cfg(tokio_unstable)))]
```

Reply:
Please fix all instances of this issue. Marking unresolved.

Reply:
Done in 7aaac8a4